services/src/datasets/postgres.rs

```diff
@@ -260,11 +252,16 @@ where
             }
         })?;
 
-        let meta_data: StaticMetaData<
-            OgrSourceDataset,
-            VectorResultDescriptor,
-            VectorQueryRectangle,
-        > = serde_json::from_value(row.get(0))?;
+        let meta_data: MetaDataDefinition = row.get("meta_data");
+
+        let MetaDataDefinition::OgrMetaData(meta_data) = meta_data else {
+            return Err(geoengine_operators::error::Error::MetaData {
+                source: Box::new(geoengine_operators::error::Error::InvalidType {
+                    expected: "OgrMetaData".to_string(),
```

Comment:
can you use the `type_name` method here?

Reply:
Maybe you can give me a hint how to do that since I don't want to construct an OgrMetaData enum variant for that.

Reply:
you can use this trait: https://github.com/geo-engine/geoengine/blob/cfa23a912140f669287176a1ad78c7994e4a87a7/operators/src/engine/operator.rs#L389

Reply:
But `OgrMetaData` is not an operator so I cannot use `OperatorName`, right?
`OperatorName` is implemented for `OgrSource`, which would yield `"OgrSource"`, but I need `"OgrMetaData"`.